isso/db/comments.py

```diff
@@ -258,7 +258,7 @@ def fetch(self, uri, mode=5, after=0, parent='any',
         # custom sanitization
         if order_by not in ['id', 'created', 'modified', 'likes', 'dislikes', 'karma']:
             order_by = 'id'
-        sql.append('ORDER BY ')
+        sql.append('ORDER BY CASE WHEN comments.parent IS NOT NULL THEN comments.id END, ')
```

Comment:
Will a `CASE WHEN .. END` immediately terminate the SQL statement, skipping over the remaining supplied arguments (`LIMIT`, ...)?
I'm not sure, could you please check?

Reply:
It shouldn't, AFAIK it just functions like the pre-existing order by, so no termination. But I'll double check it just in case.

Reply:
Does not appear to be the case:
![image](https://github.com/user-attachments/assets/57312061-df34-4009-affc-425890e20420)
![image](https://github.com/user-attachments/assets/183a5177-2bd7-4c1a-b80c-1970dd2d38ab)
(Demo page above is shown with sorting set to oldest.)